### PR TITLE
RUN-5567 Added preloadscripts for browserviews

### PR DESCRIPTION
**Pull request by pbaize**
#### Description of Change
Added Preload script support for browserview
https://testing-dashboard.openfin.co/#/app/tests/5d890db17ba87401eaf56608/edit

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] automated tests are [changed or added](https://testing-dashboard.openfin.co/#/app/dashboard)
- [ ] manual tests are [changed or added](https://github.com/openfin/test_apps)
- [ ] relevant documentation is [changed or added](https://github.com/hadoukenio/js-adapter)
- [x] PR title starts with the JIRA ticket [pull request process](https://github.com/openfin/Internal-Wiki/wiki/Pull-Request-Process)
- [x] PR release notes describe the change in a way relevant to app-developers
- [x] Link to new tests added
- [x] PR has assigned reviewers

#### Release Notes

Notes: n/a experimental

### CODE REVIEW

File: src/browser/preload_scripts.ts [hunk 1]
```diff
@@ -88,24 +89,77 @@ function loadScript(identity: Identity, preloadScript: PreloadScript): Promise<P
...
+    const frameInfo = getInfoByUuidFrame(identity);
+    let ofEntity: InjectableContext | false;
+    if (frameInfo.entityType === 'iframe') {
+        ofEntity = getEntityByUuidFrame(frameInfo.parent.uuid, frameInfo.parent.name);
+    } else {
+        ofEntity = getEntityByUuidFrame(uuid, name);
+    }
+
+    if (!ofEntity) {
+        return writeToLog('info', `setWindowPreloadState missing openfinWindow ${uuid} ${name}`);
```
Comment (MichaelMCoates) [inline src/browser/preload_scripts.ts:126]: small nitpick, should this say OpenFinWindow or View?

Comment (MichaelMCoates) [reply src/browser/preload_scripts.ts]: Ah, nvm, it's only for Windows.

Comment (MichaelMCoates) [reply src/browser/preload_scripts.ts:126]: Also should we update the name of the function?

File: src/browser/core_state.ts [hunk 2]
```diff
@@ -862,10 +852,28 @@ export interface OfView extends Identity {
...
     return ofView;
 }
+export function getEntityByUuidFrame(uuid: string, name: string): Shapes.InjectableContext | false {
+   const win = getWindowByUuidName(uuid, name);
+   if (win) {
+       return win;
+    } else {
+        return getBrowserViewByIdentity({uuid, name});
+    }
+}
```
Comment (licui3936) [inline src/browser/core_state.ts:876]: Does this function only apply for window and browserview entity types?

Comment (pbaize) [reply src/browser/core_state.ts:876]: Yes

File: src/browser/preload_scripts.ts (hunk 1 above)
Comment (pbaize) [reply src/browser/preload_scripts.ts:126]: Trying to keep this code close to the original for now.

**Pull request merged**